src/controller/ReadInteraction.h

```diff
@@ -64,7 +64,16 @@ CHIP_ERROR ReportAttribute(Messaging::ExchangeManager * exchangeMgr, EndpointId
     auto readClient =
         chip::Platform::MakeUnique<app::ReadClient>(engine, exchangeMgr, callback->GetBufferedCallback(), readParams.mReportType);
 
-    ReturnErrorOnFailure(readClient->SendRequest(readParams));
+    if (readClient->IsSubscriptionType())
+    {
+        err = readClient->SendAutoResubscribeRequest(std::move(readParams));
+        readPaths.release();
+    }
+    else
+    {
+        err = readClient->SendRequest(readParams);
+        readParams.mpAttributePathParamsList = nullptr;
+    }
 
```

Comment:
This lost the check for `err` being a success....

Reply:
yep, done, thanks